server/handlers/meshery_filter_handler.go

```diff
@@ -203,6 +203,8 @@ func (h *Handler) handleFilterPOST(
 // ```?page={page-number}``` Default page number is 0
 //
 // ```?pagesize={pagesize}``` Default pagesize is 10
+//
+// ```?visibility={visibility}``` Default visibility is public
```

Comment:
run to update the docs
```
make swagger-build
```

Reply:
Sure

Reply:
If you run that command manually, there’s no need to include its output in the PR. That command will be automatically run on merge and its output included.